src/debian/11/opt/arm64v8/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN apt-get update \
             llvm-9 \
             locales \
             make \
+            pigz \
```

Comment:
We use this image to generate profile data, we don't build on it. So we don't need to include this package here.

Reply:
```suggestion
```